docs/cloud/integrations/vscode.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Connect to Steampipe Cloud from VS Code
+sidebar_label: VS Code
+---
+## Connect to Steampipe Cloud from VS Code
+
+[VS Code](https://code.visualstudio.com/) is a source code editor with support for several languages and runtimes.
+
+The Steampipe Cloud workspace is a postgres database that can be directly connected with VS Code to query your database. The connection can be made using a query tool for PostgreSQL databases available from the extension marketplace.
+
+The [Connect](/docs/cloud/integrations/overview) tab for your workspace provides the details you need to connect VS Code to Steampipe Cloud.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
+<img src="/images/docs/cloud/steampipe-cloud-connect-details.jpg" />
+</div>
+
+Once Steampipe Cloud is successfully connected, you can explore the tables provided by the Steampipe plugins, run queries and build reports.
```

Comment:
I wanted to insert that section in a suggested edit, but that breaks because the suggested edit is done with triple-backticks and so can't seem to include a nested triple-backtick thing?

Reply:
Rectified. 

Yes, it would not show up as required but would come as separate lines of code in the suggested Edit. Which is pretty weird since it allows that to be in the code.